phoenix-core/src/test/java/org/apache/phoenix/index/PrepareIndexMutationsForRebuildTest.java

```diff
@@ -0,0 +1,489 @@
+package org.apache.phoenix.index;
```

Comment:
@priyankporwal @kadirozde @gjacoby126 @swaroopak , can you help to review when you get a chance. 
For some reason, it looks like I don't have permission to add reviewers.